<?xml version="1.0" encoding="UTF-8"?>
<Examples>
  <Example>
    <Log>C#/NancyFx@Nancy/failed/375658615.log</Log>
    <Keywords>error, , </Keywords>
    <Category>0</Category>
    <Chunk>An error occured while installing Cake.</Chunk>
  </Example>
  <Example>
    <Log>C#/NancyFx@Nancy/failed/375800180.log</Log>
    <Keywords>FAIL, Failure, Error</Keywords>
    <Category>1</Category>
    <Chunk>Nancy.Testing.Tests.BrowserFixture.Should_return_JSON_serialized_form [FAIL]

[m[37m      Assert.Equal() Failure

[m[37m      Expected: 1

[m[37m      Actual:   0

[m[30;1m      Stack Trace:

[m[37m        /Users/travis/build/NancyFx/Nancy/test/Nancy.Testing.Tests/BrowserFixture.cs(445,0): at Nancy.Testing.Tests.BrowserFixture.Should_return_JSON_serialized_formd__24.MoveNext()

[m[37m        --- End of stack trace from previous location where exception was thrown ---

[m[37m           at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()

[m[37m           at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)

[m[37m        --- End of stack trace from previous location where exception was thrown ---

[m[37m           at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()

[m[37m           at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)

[m[37m        --- End of stack trace from previous location where exception was thrown ---

[m[37m           at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()

[m[37m           at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)</Chunk>
  </Example>
  <Example>
    <Log>C#/NancyFx@Nancy/failed/377324945.log</Log>
    <Keywords>FAIL, Failure, Error</Keywords>
    <Category>1</Category>
    <Chunk>Nancy.Testing.Tests.BrowserFixture.Should_return_JSON_serialized_form [FAIL]
[m[37m      Assert.Equal() Failure
[m[37m      Expected: 1
[m[37m      Actual:   0
[m[30;1m      Stack Trace:
[m[37m        /home/travis/build/NancyFx/Nancy/test/Nancy.Testing.Tests/BrowserFixture.cs(445,0): at Nancy.Testing.Tests.BrowserFixture.Should_return_JSON_serialized_formd__24.MoveNext()
[m[37m        --- End of stack trace from previous location where exception was thrown ---
[m[37m           at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
[m[37m           at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
[m[37m        --- End of stack trace from previous location where exception was thrown ---
[m[37m           at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
[m[37m           at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
[m[37m        --- End of stack trace from previous location where exception was thrown ---
[m[37m           at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
[m[37m           at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)</Chunk>
  </Example>
  <Example>
    <Log>C#/NancyFx@Nancy/failed/382556234.log</Log>
    <Keywords>FAIL, Failure, Error</Keywords>
    <Category>1</Category>
    <Chunk>Nancy.Testing.Tests.BrowserFixture.Should_return_JSON_serialized_querystring [FAIL]
[m[37m      Assert.Equal() Failure
[m[37m      Expected: 1
[m[37m      Actual:   0
[m[30;1m      Stack Trace:
[m[37m        /home/travis/build/NancyFx/Nancy/test/Nancy.Testing.Tests/BrowserFixture.cs(473,0): at Nancy.Testing.Tests.BrowserFixture.Should_return_JSON_serialized_querystringd__25.MoveNext()
[m[37m        --- End of stack trace from previous location where exception was thrown ---
[m[37m           at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
[m[37m           at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
[m[37m        --- End of stack trace from previous location where exception was thrown ---
[m[37m           at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
[m[37m           at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
[m[37m        --- End of stack trace from previous location where exception was thrown ---
[m[37m           at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
[m[37m           at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)</Chunk>
  </Example>
  <Example>
    <Log>C#/NancyFx@Nancy/failed/386536177.log</Log>
    <Keywords>FAIL, Failure, Error</Keywords>
    <Category>1</Category>
    <Chunk>Nancy.Testing.Tests.BrowserFixture.Should_return_JSON_serialized_querystring [FAIL]
[m[37m      Assert.Equal() Failure
[m[37m      Expected: 1
[m[37m      Actual:   0
[m[30;1m      Stack Trace:
[m[37m        /home/travis/build/NancyFx/Nancy/test/Nancy.Testing.Tests/BrowserFixture.cs(473,0): at Nancy.Testing.Tests.BrowserFixture.Should_return_JSON_serialized_querystringd__25.MoveNext()
[m[37m        --- End of stack trace from previous location where exception was thrown ---
[m[37m           at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
[m[37m           at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
[m[37m        --- End of stack trace from previous location where exception was thrown ---
[m[37m           at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
[m[37m           at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
[m[37m        --- End of stack trace from previous location where exception was thrown ---
[m[37m           at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
[m[37m           at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)</Chunk>
  </Example>
  <Example>
    <Log>C#/NancyFx@Nancy/failed/405164163.log</Log>
    <Keywords>FAIL, Failure, Error</Keywords>
    <Category>1</Category>
    <Chunk>Nancy.Testing.Tests.BrowserFixture.Should_return_JSON_serialized_form [FAIL]
[m[37m      Assert.Equal() Failure
[m[37m      Expected: 1
[m[37m      Actual:   0
[m[30;1m      Stack Trace:
[m[37m        /home/travis/build/NancyFx/Nancy/test/Nancy.Testing.Tests/BrowserFixture.cs(445,0): at Nancy.Testing.Tests.BrowserFixture.Should_return_JSON_serialized_formd__24.MoveNext()
[m[37m        --- End of stack trace from previous location where exception was thrown ---
[m[37m           at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
[m[37m           at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
[m[37m        --- End of stack trace from previous location where exception was thrown ---
[m[37m           at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
[m[37m           at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
[m[37m        --- End of stack trace from previous location where exception was thrown ---
[m[37m           at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
[m[37m           at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)</Chunk>
  </Example>
  <Example>
    <Log>C#/NancyFx@Nancy/failed/421691530.log</Log>
    <Keywords>FAIL, Failure, Error</Keywords>
    <Category>1</Category>
    <Chunk>Nancy.Testing.Tests.BrowserFixture.Should_return_JSON_serialized_form [FAIL]

[m[37m      Assert.Equal() Failure

[m[37m      Expected: 1

[m[37m      Actual:   0

[m[30;1m      Stack Trace:

[m[37m        /Users/travis/build/NancyFx/Nancy/test/Nancy.Testing.Tests/BrowserFixture.cs(445,0): at Nancy.Testing.Tests.BrowserFixture.Should_return_JSON_serialized_formd__24.MoveNext()

[m[37m        --- End of stack trace from previous location where exception was thrown ---

[m[37m           at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()

[m[37m           at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)

[m[37m        --- End of stack trace from previous location where exception was thrown ---

[m[37m           at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()

[m[37m           at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)

[m[37m        --- End of stack trace from previous location where exception was thrown ---

[m[37m           at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()

[m[37m           at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)</Chunk>
  </Example>
  <Example>
    <Log>C#/NancyFx@Nancy/failed/433641108.log</Log>
    <Keywords>Failed, FATAL, failed</Keywords>
    <Category>2</Category>
    <Chunk>/home/travis/.rvm/gems/ruby-1.9.3-p551/gems/albacore-1.0.0.rc.2/lib/albacore/support/failure.rb:12:in `fail_with_message'
/home/travis/.rvm/gems/ruby-1.9.3-p551/gems/albacore-1.0.0.rc.2/lib/albacore/xunittestrunner.rb:48:in `block in execute'
/home/travis/.rvm/gems/ruby-1.9.3-p551/gems/albacore-1.0.0.rc.2/lib/albacore/xunittestrunner.rb:44:in `execute'
/home/travis/.rvm/gems/ruby-1.9.3-p551/gems/albacore-1.0.0.rc.2/lib/albacore/support/createtask.rb:24:in `block in xunit'
/home/travis/.rvm/gems/ruby-1.9.3-p551/gems/rake-12.2.1/exe/rake:27:in `top (required)&gt;'
/home/travis/.rvm/gems/ruby-1.9.3-p551/bin/ruby_executable_hooks:15:in `eval'
/home/travis/.rvm/gems/ruby-1.9.3-p551/bin/ruby_executable_hooks:15:in `main&gt;'</Chunk>
  </Example>
  <Example>
    <Log>C#/NancyFx@Nancy/failed/461393228.log</Log>
    <Keywords>FAILED, error, Error</Keywords>
    <Category>3</Category>
    <Chunk>Extensions/CollectionExtensions.cs(20,40): error CS0103: The name 'key' does not exist in the current context [/home/travis/build/NancyFx/Nancy/src/Nancy/Nancy.csproj]
Extensions/CollectionExtensions.cs(21,17): error CS0103: The name 'key' does not exist in the current context [/home/travis/build/NancyFx/Nancy/src/Nancy/Nancy.csproj]
Extensions/CollectionExtensions.cs(23,24): error CS0029: Cannot implicitly convert type 'string[]' to 'System.Collections.Generic.IDictionarystring, System.Collections.Generic.IEnumerablestring&gt;&gt;' [/home/travis/build/NancyFx/Nancy/src/Nancy/Nancy.csproj]
Extensions/CollectionExtensions.cs(25,20): error CS0029: Cannot implicitly convert type 'string[]' to 'System.Collections.Generic.IDictionarystring, System.Collections.Generic.IEnumerablestring&gt;&gt;' [/home/travis/build/NancyFx/Nancy/src/Nancy/Nancy.csproj]</Chunk>
  </Example>
  <Example>
    <Log>C#/NancyFx@Nancy/failed/533080176.log</Log>
    <Keywords>FAIL, Failure, Error</Keywords>
    <Category>1</Category>
    <Chunk>Nancy.Testing.Tests.BrowserFixture.Should_return_JSON_serialized_form [FAIL]

[m[37m      Assert.Equal() Failure

[m[37m      Expected: 1

[m[37m      Actual:   0

[m[30;1m      Stack Trace:

[m[37m        /Users/travis/build/NancyFx/Nancy/test/Nancy.Testing.Tests/BrowserFixture.cs(445,0): at Nancy.Testing.Tests.BrowserFixture.Should_return_JSON_serialized_formd__24.MoveNext()

[m[37m        --- End of stack trace from previous location where exception was thrown ---

[m[37m           at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()

[m[37m           at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)

[m[37m        --- End of stack trace from previous location where exception was thrown ---

[m[37m           at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()

[m[37m           at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)

[m[37m        --- End of stack trace from previous location where exception was thrown ---

[m[37m           at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()

[m[37m           at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)</Chunk>
  </Example>
</Examples>
